getstarted.rst

```diff
@@ -37,7 +37,7 @@ For Windows
         - **Append** (do not delete any text that exists) the following text at the end of the line: ``;C:\Python34;C:\Python34\Scripts;``
         - hit ``OK`` until out of all configuration windows
     - **Create a** ``python3`` **shortcut to work with the rest of this tutorial** 
-    *You may be able to skip this and just use ``python`` instead of ``python3`` for the rest of this tutorial*
+    **You may be able to skip this and just use ``python`` instead of ``python3`` for the rest of this tutorial
```

Comment:
What's up with changing the * here. Wrapped *like this* is supposed to italicize the text. What does two ** at the front mean?